pokemongo_bot/cell_workers/pokemon_catch_worker.py

```diff
@@ -324,8 +356,12 @@ def should_release_pokemon(self, pokemon_name, cp, iv, response_dict):
 
         if release_config.get('always_release', False):
             return True
-
-        release_cp = release_config.get('release_below_cp', 0)
+	
+	if release_config.get('keep_best_cp', True):
```

Comment:
Maybe we would want the default value to be False, so nothing changes for users who pull but don't update their release_config.json file?